content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using API Gateway for Data Services Cloud Console
+date: 2021-07-29T14:28:17.779Z
+featuredBlog: false
+priority: 1
+author: Ron Dharma
+authorimage: /img/dscc-icon.png
+thumbnailimage: ""
+tags:
+  - data-services-cloud-console
+---
+## Authentication Process to obtain the Access Token
+
+Data Services Cloud Console public API relies on OAuth 2.0 third party authorization framework on behalf of the resource owner (DSCC user) for security. User starts by authenticating into HPE GreenLake which is accommodated by the Identity Provider (validated by username, password, MFA or SSO). Using the API Gateway menu in HPE GreenLake customer registers their application to obtain the OAuth 2.0 API client credentials. This association allows user to obtain the Access Token.
```

Comment:
Suggestion: The Data Services Cloud Console public API relies on the open-standard OAuth 2.0 third party framework.... 

Also, when you mention " the customer registers their application", it is not clear for the readers what is the "application". do you mean "the customer registers their Data Services (Common) Cloud Console application"? 

I believe it would make sense to add in the introductory paragraph a little bit more context for the readers to better understand "what is the Data Services Cloud Console app" used for, and a gist about the concept of the "region" (if such concept exists in DSCC) where the customer's app is installed.